git-tutorial.markdown

```diff
@@ -98,11 +98,77 @@ If you have your old installation under `~/alice` then we suggest you use a
 different directory to avoid confusion.
 
 
+### I don't use aliBuild
```

Comment:
I agree with @jgrosseo, experience tells us that some users will ask if this is not visible in the main documentation.